controllers/mirrorpeer_controller.go

```diff
@@ -59,6 +61,27 @@ const hubRecoveryLabel = "cluster.open-cluster-management.io/backup"
 //+kubebuilder:rbac:groups=addon.open-cluster-management.io,resources=managedclusteraddons;clustermanagementaddons,verbs=*
 //+kubebuilder:rbac:groups=addon.open-cluster-management.io,resources=managedclusteraddons/status,verbs=*
 
+// Checks whether a string is contained within a slice
+func contains(slice []string, s string) bool {
```

Comment:
This function is already covered in rook_secret_handler.go 
Please move them to the utils and use it.